python/_tskitmodule.c

```diff
@@ -1981,6 +2125,45 @@ EdgeTable_add_row(EdgeTable *self, PyObject *args, PyObject *kwds)
     return ret;
 }
 
+static PyObject *
+EdgeTable_update_row(EdgeTable *self, PyObject *args, PyObject *kwds)
+{
+    PyObject *ret = NULL;
+    int err;
+    tsk_id_t row_index = -1;
+    double left = 0.0;
+    double right = 1.0;
+    int parent;
```

Comment:
might as well save a big of vspace here, ``int parent, child;``. Same for left and right, we're not using the values we're setting, so no point in setting them.

Reply:
Yep, done in a couple of the methods